backend/src/main/scala/cromwell/EngineFunctions.scala

```diff
@@ -0,0 +1,24 @@
+package cromwell
```

Comment:
Now that we have the subprojects should we be better about full package names (yuck) - e.g. `org.broadinstitute.blah.blah.backend.cromwell`? Specifically my concern is how all of this works within the fat jar